hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/workflows/QueryContext.java

```diff
@@ -29,7 +29,7 @@
 /**
  * Context of a single query. Contains all query specific information.
  */
-public interface QueryContext {
+public interface QueryContext extends OperationContext {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Why do we need to consider `QueryContext`? Isn't `HandleContext` enough?

Reply:
I thought `ContextQueryProcessor` calls `ProxyWorldUpdater.commit()` and `ContextTransactionProcessor` calls `RootProxyWorldUpdater.commit()` but I might be wrong.

Having the more general `OperationContext` interface looks good.